### ISSUE TITLE
symbol lookup error

### ISSUE THREAD

**Issue by tauriel27** [jintaoluo/presto2_on_gpu#1]
readfile:symbol lookup error: /home/dataology/astrosoft/presto/lib/libpresto.so: undefined symbol: _ZNSt8ios_base4InitD1Ev

when I finished "make" in /presto/src and I tried some presto cmd, but I got this error 
libpresto.so exists in /home.../presto/lib

if I choose to use cpu in Makefile ,I didn't got this error

**Comment by jintaoluo** [comment]
Sounds like you have multiple PRESTO on your system?

The suggestion I'd like to make is ensure the system varible 'PRESTO' is pointed to the PRESTO you want to use. For example, you are working with the PRESTO located at /presto, the varible 'PRESTO' should be pointed to '/presto'. With ubuntu and centos I will use 'export PRESTO=/presto' to do this job. It is just to ensure you compile and the use the 'SAME' PRESTO.

Hope it help :)

**Comment by tauriel27** [comment]
yes ,I have installed scottransom/presto on my system and it works. 
But Scott's presto is renamed presto1 and presto2_on_gpu is renamed presto,then I compile this "new presto",if I use cpu, it works, if gpu,it doesn't work. so I guess maybe I have something wrong with cuda configuration? This is my /etc/environment
PATH="/../presto/bin:/usr/local/cuda-8.0/bin”
LD_LIBRARY_PATH=”/usr/local/cuda-8.0/lib64:/../presto/lib”
（it doesn't show errors when I use "make" in /presto/src,and in Makefile line 22 : use_cuda = yes# yes/no )

**Comment by jintaoluo** [comment]
I guess all the PRESTO related variables, including PATH and LD_LIBRARY_PATH, should be modified to represent what you are actually working on. Then when you run a PRESTO program from the GPU compilation, it can find and use the 'RIGHT' libraries that are compiled from the GPU version as well. 

You may take my environment as the reference:
]$ echo $PATH
/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/jluo/bin:/usr/local/cuda-6.5/bin:/home/jluo/PSRHOME/bin:/home/jluo/my_soft_devl/presto-master_on_GPU/bin
]$ echo $LD_LIBRARY_PATH 
:/usr/local/cuda-6.5/lib64:/usr/lib:/home/jluo/PSRHOME/lib:/home/jluo/PSRHOME/pgplot_build:/home/jluo/my_soft_devl/presto-master_on_GPU/lib

With this configuration, the GPU version of PRESTO works for me.
